### PR TITLE
Fixes #107: added support to create unique id for each build

### LINKED ISSUE #107: create unique folder for each documentation build
### This is a
- [ ] Bug report
- [x] Feature request

### Current behaviour

All the generated documentation are kept in email folder and it's respective project name. since we not verifying email, we don't know how much the email is authentic and also if two person generate doc with same email and same project name, there is a huge chance of missing the documentation
### Expected behaviour

All the build documentation should be kept on unique folder

### Steps to reproduce
1.
2.
3.

### PR DESCRIPTION
**Pull request by sch00lb0y**
## Description

added support to have unique id for each documentation build
## Related Issue

close #107 
## Types of changes

- [ ] Bug fix 
- [x] New feature

## Checklist:

- [x] There is a corresponding issue for this pull request.
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

**Comment by sch00lb0y** [comment]
@pri22296 @imujjwal96 please review this

File: WebUI/routes/index.js [hunk 1]
```diff
@@ -16,32 +16,33 @@ router.post("/generate", function (req, res, next) {
   var docPath = req.body.doc_path;
   var projectName = req.body.project_name;
   var version = req.body.version;
-
+  var uniqueId = Date.now()
```
Comment (niranjan94) [inline WebUI/routes/index.js]: You could use [UUID](https://tools.ietf.org/html/rfc4122) v4 instead of this to get a more reliable unique ID.

Comment (sch00lb0y) [inline WebUI/routes/index.js]: I have changed as you suggested.
Thanks

File: WebUI/generate.sh [hunk 2]
```diff
@@ -12,6 +12,7 @@ do
  o) PROJECTNAME=${OPTARG};;
  v) VERSION=${OPTARG};;
  m) EMAIL=${OPTARG};;
+ u) UNIQUEID=${OPTARG};;
```
Comment (imujjwal96) [inline WebUI/generate.sh]: why have you not specified the `u` tag in line 4?

Comment (sch00lb0y) [inline WebUI/generate.sh]: I have added the tag. I tested also. Thanks

File: WebUI/generate.sh [hunk 3]
```diff
@@ -41,9 +42,9 @@ BUILD_DIR=$(pwd)/yaydoctemp
 
 cd $ROOT_DIR
 
-cp -a ../../../scripts/ $BUILD_DIR/
-cp -a ../../../templates/ $BUILD_DIR/
-cp -a ../../../requirements.txt $BUILD_DIR/
+cp -a ../../../../scripts/ $BUILD_DIR/
+cp -a ../../../../templates/ $BUILD_DIR/
+cp -a ../../../../requirements.txt $BUILD_DIR/
```
Comment (imujjwal96) [inline WebUI/generate.sh]: why was this change needed?

Comment (sch00lb0y) [inline WebUI/generate.sh]: in order to copy it has to go one more depth. previously it was going upto `WebUI`, so i have added one more depth

Comment (imujjwal96) [inline WebUI/generate.sh]: This change was introduced by you in the last PR. So you're saying that you made the changes and sent a PR without even testing it?

Comment (sch00lb0y) [inline WebUI/generate.sh]: sorry,I thought it works.

Comment (imujjwal96) [inline WebUI/generate.sh]: You don't need to. It's not all your fault. I was supposed to test the PR. Saw that the exit code returned by the script was 0 and thought it would be correct. 
It was later when I checked the logs properly, i found the bug.
